ci-operator/step-registry/gather/azure-resource/gather-azure-resource-commands.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+export PATH=$PATH:/tmp
+mkdir -p /tmp
+
+echo "$(date -u --rfc-3339=seconds) - Installing tools..."
+
+# Make sure jq is installed
+if ! command -v jq; then
```

Comment:
I think we do not need this step based on the step is using upi-installer image.